keyboards/handwired/scottokeebs/scottolong/info.json

```diff
@@ -0,0 +1,72 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "ScottoLong",
+    "maintainer": "joe-scotto",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP6", "GP7", "GP8", "GP9", "GP10", "GP11", "GP12", "GP13", "GP14", "GP15"],
+        "rows": ["GP2", "GP3", "GP4", "GP5"]
+    },
+    "processor": "RP2040",
+    "url": "https://scottokeebs.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0011",
+        "vid": "0xFEED"
```

Comment:
Vendor ID cannot be FEED for VIA support.

Reply:
Resolved in [f3bffe3](https://github.com/qmk/qmk_firmware/pull/20938/commits/f3bffe37a89f128dd7692878a19ab63f2a6ff649). Board has Vial on my site for GUI support, VIA not needed.